integration

```diff
@@ -185,6 +185,7 @@
   "ClusterM/skykettle-ha",
   "CM000n/qss",
   "Cmajda/ha_golemio",
+  "cnecrea/apanova",
```

Comment:
For a specific country, and the released hacs.json file is missing the country key.